docs/gettingstarted/installation.rst

```diff
@@ -263,19 +263,18 @@ You'll probably want to setup at least two nodes.
    At the time of writing, the only supported Fedora 20 kernel on DigitalOcean is ``Fedora 20 x64 vmlinuz-3.16.6-203.fc20.x86_64``.
```

Comment:
It seems that `Fedora 20 x64 vmlinuz-3.17.8-200.fc20.x86_64` is chosen by `set_latest_droplet_kernel` so one of the two is wrong. I guess that this document needs updating.